### PR TITLE
increases max optical dup distance to  2x recommended for novaseq

### PR DESCRIPTION
**Pull request by bwlang**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bwlang** [comment]
quick fix snowballed a bit...

File: tools/picard/picard_CollectBaseDistributionByCycle.xml [hunk 1]
```diff
@@ -59,8 +59,8 @@
   </inputs>
 
   <outputs>
-    <data format="tabular" name="outFile"/>
-    <data format="pdf" name="pdfFile"/>
+    <data format="tabular" name="outFile" label="${tool.name} on ${on_string}: CollectBaseDistributionByCycle tabular output"/>
+    <data format="pdf" name="pdfFile"  label="${tool.name} on ${on_string}: CollectBaseDistributionByCycle PDF output"/>
```
Comment (bernt-matthias) [inline tools/picard/picard_CollectBaseDistributionByCycle.xml:63]:
```suggestion
    <data format="tabular" name="outFile" label="${tool.name} on ${on_string}: tabular output"/>
    <data format="pdf" name="pdfFile"  label="${tool.name} on ${on_string}: PDF output"/>
```

`${tool.name}` is already `CollectBaseDistributionByCycle`

File: tools/picard/picard_CollectInsertSizeMetrics.xml [hunk 2]
```diff
@@ -72,8 +72,8 @@
   </inputs>
 
   <outputs>
-    <data format="tabular" name="outFile"/>
-    <data format="pdf" name="histFile"/>
+    <data format="tabular" name="outFile" label="${tool.name} on ${on_string}: CollectBaseDistributionByCycle tabular output"/>
+    <data format="pdf" name="histFile" label="${tool.name} on ${on_string}: CollectBaseDistributionByCycle PDF output"/>
```
Comment (bernt-matthias) [inline tools/picard/picard_CollectInsertSizeMetrics.xml:76]:
```suggestion
    <data format="tabular" name="outFile" label="${tool.name} on ${on_string}: tabular output"/>
    <data format="pdf" name="pdfFile"  label="${tool.name} on ${on_string}: PDF output"/>
```

see above + wrong tool name

File: tools/picard/picard_MergeBamAlignment.xml [hunk 3]
```diff
@@ -191,7 +191,7 @@
       <param name="max_insertions_or_deletions" value="1"/>
       <param name="read1_trim" value="0"/>
       <param name="read2_trim" value="0"/>
-      <param name="orientation" value="FR"/>
+      <param name="orientations" value="FR"/>
```
Comment (bernt-matthias) [inline tools/picard/picard_MergeBamAlignment.xml:194]: good finding xref https://github.com/galaxyproject/tools-iuc/issues/4095

Also started working on this recently  https://github.com/galaxyproject/tools-iuc/pull/5103 but got distracted :( .. maybe you want to check if there is anything else of use?

Comment (bwlang) [reply tools/picard/picard_MergeBamAlignment.xml:194]: after a quick look i see some other important improvements, but i'm out of time  for the moment...

Comment (bernt-matthias) [reply tools/picard/picard_MergeBamAlignment.xml:194]: Lets go small steps then :)

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**